node/.env.example

```diff
@@ -10,5 +10,7 @@ PROVIDER_URL=
 #not required if wallets table is not empty
 REPORTER_PK=
 
+SUBMISSION_PROXY_CONTRACT=
```

Comment:
@nick-bisonai Let's keep in mind that in future we might need more than one `SubmissionProxy` contract 🙏